### DISCUSSION TITLE
About Drawer attribute "extra"关于抽屉的extra属性问题

### DISCUSSION THREAD

**Discussion by YuriPLSk** [ant-design/ant-design#33428] [Q&A]
我仿照官网写的抽屉的extra属性为什么不渲染呢？
![image](https://user-images.githubusercontent.com/49810195/147457701-b3f7dc83-2dcb-4b66-b03d-4f703c3dcda4.png)
![image](https://user-images.githubusercontent.com/49810195/147457756-9a7d63fd-cec0-46f8-8dc2-bde5ddaad316.png)
![image](https://user-images.githubusercontent.com/49810195/147457795-4f0f181d-1e24-4857-ba86-c86d4c736646.png)
求各位指导

**Comment by liuyib** [comment]
@YuriPLSk 控制台查看元素，看看你指定的元素到底渲染了没，如果有对应的 DOM，那就是你的样式写的有问题；如果没有对应DOM，那再另说
